server/src/internalClusterTest/java/org/elasticsearch/cluster/allocation/ClusterRerouteIT.java

```diff
@@ -199,6 +200,7 @@ public void testRerouteWithAllocateLocalGateway_enableAllocationSettings() throw
     }
 
     public void testDelayWithALargeAmountOfShards() throws Exception {
+        assumeFalse("Windows can be too slow on handling larget amount of cluster state updates", Constants.WINDOWS);
```

Comment:
As [commented](https://github.com/elastic/elasticsearch/issues/137384#issuecomment-3470761846) on the issue, one alternative is to extend the timeout on Windows. But it's already 2min and I don't particularly like to increase it more and it does not feel all that useful to keep the test for Windows either.